packages/gatsby-theme-newrelic/src/utils/createTessen.js

```diff
@@ -68,6 +72,8 @@ const tessenAction =
 
     const customerId = getCookie('ajs_user_id');
     const anonymousId = getCookie('ajs_anonymous_id');
+    const loggedIn =
+      getSavedLoggedInStatus() ?? (await checkIfUserLoggedIn(config.product));
```

Comment:
is it fine to use config.product here to populate the NewRelic-Requesting-Services header?

Reply:
my two cents: i think we should probably add a new theme option explicitly for this. maybe call it something like `nerdgraphRequestingServicesHeader` so it's super clear what it's for and because it may require a specific format (looks like slug, separated by hyphens).

that option would be passed to the theme via a site's `gatsby-config.js` in the `@newrelic/gatsby-theme-newrelic` plugin's `options`. probably a new field there, because `newrelic` is specific to the browser agent configs.

after a bit of digging, i don't _think_ the service key passed along in this header requires any other configuring via service gateway or nerdgraph.

Reply:
if we go that route, we'd also want to add doc on that new option here:

https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#options

Reply:
done https://github.com/newrelic/gatsby-theme-newrelic/pull/826/files#diff-b0b7cd4f1caa98b028107f2cd31a286d39e7a926edecd9cca4bd3150fa1790e2R597